hyok_configuration.go

```diff
@@ -0,0 +1,311 @@
+package tfe
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+)
+
+type HYOKConfigurations interface {
+	List(ctx context.Context, organization string, options *HYOKConfigurationsListOptions) (*HYOKConfigurationsList, error)
+
+	Create(ctx context.Context, organization string, options HYOKConfigurationsCreateOptions) (*HYOKConfiguration, error)
+
+	Read(ctx context.Context, hyokID string, options *HYOKConfigurationsReadOptions) (*HYOKConfiguration, error)
+
+	Update(ctx context.Context, hyokID string, options HYOKConfigurationsUpdateOptions) (*HYOKConfiguration, error)
+
+	Delete(ctx context.Context, hyokID string) error
+
+	Test(ctx context.Context, hyokID string) error
```

Comment:
It could be nice to add a comment explaining how this function behaves: i.e if there is no error, than a successful test was performed.